src/ansys/fluent/core/examples/downloads.py

```diff
@@ -131,3 +132,11 @@ def download_file(
 
     url = _get_file_url(filename, directory)
     return _retrieve_file(url, filename, save_path, return_only_filename)
+
+
+def path(filename: str):
+    file_path = str(Path(pyfluent.EXAMPLES_PATH) / filename)
+    if os.path.isfile(file_path):
```

Comment:
If you want to use the `Path` object from the previous line it also has an equivalent method.

Reply:
Thank you.